_functions_tomcat.sh

```diff
@@ -185,7 +185,11 @@ do_configure_tomcat_datasources() {
   case ${DEPLOYMENT_DATABASE_TYPE} in
     MYSQL|DOCKER_MYSQL|DOCKER_MARIADB)
       # Patch to reconfigure server.xml for database
-      find_instance_file DB_SERVER_PATCH "${ETC_DIR}/${DEPLOYMENT_APPSRV_TYPE}${DEPLOYMENT_APPSRV_VERSION:0:1}" "server-mysql.xml.patch" "${DB_SERVER_PATCH_PRODUCT_NAME}"
+      if [[ "${PRODUCT_VERSION}" =~ ^(5.2) ]]; then
```

Comment:
if I am right, there is no need to add such if condition because the function `find_instance_file` use the `${DB_SERVER_PATCH_PRODUCT_NAME}` parameter to find the best patch

Please double check

Reply:
I got it now, I switched to use 'find_instance_file' to automatically get the right patch.